weave/upload.py

```diff
@@ -175,6 +175,23 @@ def __init__(
         and keep the pantry organized.
         """
 
+        # if there is no files
+        # if parent uids are provided
+        # if metadata is provided
+        # then it is a metadata only basket
+
+
+        parent_uids = kwargs.get("parent_ids", "no parent ids")
+        basket_metadata = kwargs.get("metadata", "no metadata")
```

Comment:
I feel like it is not good practice to have two different data types in the kwargs.get function. Maybe you could change "no parent ids" to just be [].

Reply:
You're right, I changed it to clean it up